src/main/java/ru/svgogin/service/spark/exception/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,20 @@
+package ru.svgogin.service.spark.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.CONFLICT, reason = "This company already exists")
+public class EntityAlreadyExistsException extends RuntimeException {
+  private final String message;
+
+  public EntityAlreadyExistsException(String message) {
+    this.message = message;
```

Comment:
```suggestion
    super(message);
```
And you don't need to override `message` and you can remove field `message`

Reply:
Done